frontend/taipy-gui/src/components/Taipy/index.ts

```diff
@@ -43,6 +43,7 @@ import Table from "./Table";
 import TaipyStyle from "./TaipyStyle";
 import Toggle from "./Toggle";
 import TreeView from "./TreeView";
+import Notification from "./Notification";
```

Comment:
In the nearest possible future, we will have to rename 'Notification' to 'Alert' for homogeneity (although programmers don't really care).

Reply:
Hi @FabienLelaquais thanks for your review, but `Alert.tsx` already exists.